File: typical90/typical90_v.py
from math import gcd


def main():
    n = gcd(A, gcd(B, C))
    ans = (A + B + C) // n - 3
    return print(ans)


if __name__ == '__main__':
    A, B, C = map(int, input().split())
    main()

File: ur_typical90/typical90_v.cpp
#include <bits/stdc++.h>
using namespace std;
using ll = long long;

int main() {
    ll a, b, c;
    cin >> a >> b >> c;

    ll n = gcd(a, gcd(b, c));
    ll ans = (a + b + c) / n - 3;
    cout << ans << endl;

    return 0;
}
